### PR TITLE
Investment journey block

### PR DESCRIPTION
**Pull request by Katiasun**

### CODE REVIEW

File: src/components/InvestmentJourneyBlock/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  .tittleReadyFor {
+    font-size: 48px;
+  }
+
+  .paragraphWithOur {
+    font-size: 18px;
+    width: 572px;
+  }
+
+  .upperTriangle,
```
Comment (Todmy) [inline src/components/InvestmentJourneyBlock/styles.module.css:81]: this class is not used

File: src/components/InvestmentJourneyBlock/index.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <div className={styles.itemCard}>
+          <h5 className={styles.titleCard}>
+            {intl.formatMessage({ id: "InvestmentJourneyBlock.StartRightNow" })}
+          </h5>
+          <p className={styles.textCard}>
+            {intl.formatMessage({ id: "InvestmentJourneyBlock.AndGetReady" })}
+          </p>
+        </div>
+      </div>
+      <img
```
Comment (Todmy) [inline src/components/InvestmentJourneyBlock/index.jsx:58]: is it redundant

**Review by Todmy — CHANGES_REQUESTED** [review]

File: src/components/InvestmentJourneyBlock/styles.module.css (hunk 1 above)
Comment (Katiasun) [reply src/components/InvestmentJourneyBlock/styles.module.css:81]: delete this class

File: src/components/InvestmentJourneyBlock/index.jsx (hunk 2 above)
Comment (Katiasun) [reply src/components/InvestmentJourneyBlock/index.jsx:58]: delete img also

**Review by Todmy — APPROVED** [review]

**Pull request merged**